src/pr-info.ts

```diff
@@ -256,7 +256,7 @@ export async function deriveStateForPR(
         headCommitAbbrOid: headCommit.abbreviatedOid,
         headCommitOid: headCommit.oid,
         mergeIsRequested: !!prInfo.comments.nodes
-            && usersSayReadyToMerge(noNulls(prInfo.comments.nodes),
+            && usersSayReadyToMerge(prInfo, noNulls(prInfo.comments.nodes),
                                     dangerLevel.startsWith("Scoped") ? [author, ...allOwners] : [author]),
```

Comment:
This is not the problem: note that `author` is always included in the list...  I'll re-fetch the fixture for this pr and will try to edit away recent interactions to see if it fails.

Reply:
Cool, 👍  then it is likely that the webhook wasn't being picked up